Lorem Ipsum XR Translator/Assets/Scripts/SpeechToText/AzureSpeechToTextButton.cs

```diff
@@ -0,0 +1,47 @@
+using UnityEngine;
+using Microsoft.CognitiveServices.Speech;
+using Microsoft.MixedReality.Toolkit.UI;
+using TMPro;
+using Config;
+
+public class AzureSpeechToTextButton : MonoBehaviour
```

Comment:
This is a good start for testing functionality. I would like to see this converted to a service like our other Azure service calls before it is merged.